packages/evm/src/interpreter.ts

```diff
@@ -184,40 +190,54 @@ export class Interpreter {
   }
 
   async run(code: Uint8Array, opts: InterpreterOpts = {}): Promise<InterpreterResult> {
-    if (!this.common.isActivatedEIP(3540) || code[0] !== EOF.FORMAT) {
+    if (!this.common.isActivatedEIP(3540) || code[0] !== FORMAT) {
       // EIP-3540 isn't active and first byte is not 0xEF - treat as legacy bytecode
       this._runState.code = code
     } else if (this.common.isActivatedEIP(3540)) {
-      if (code[1] !== EOF.MAGIC) {
+      if (code[1] !== MAGIC) {
```

Comment:
Just occurred to me, will we ever have other kinds of magic (e.g. authcall tx call data MAGIC bytes or something like that)?  Would hate to have to rename this again later if we end up with multiple types of magic bytes to check for.

Reply:
I have used the terminology of EOF, but you are right, the current 7702 spec essentially uses the magic of 0x01 to designate some particular behavior :sweat_smile: